src/com/company/GameService.java

```diff
@@ -0,0 +1,36 @@
+package com.company;
+
+public class GameService {
+
+  IBoard iBoard;
+  int Row = 3;
+  int Column = 3;
+
+  public GameService() {
+    this.iBoard = new Board(Row, Column);
+  }
+
+  public boolean nextMove(int row, int column, Players player) {
+
+    iBoard.setPiece(row, column, player);
+    iBoard.printBoard();
+    if (iBoard.checkDiagonalWin(player)) {
+      System.out.println(player.getName() + " won the game");
+    }
+    if (iBoard.checkHorizontalWin(player)) {
```

Comment:
Always use else. Your code will print the same thing twice in certain cases based on the current code.

Reply:
ok